### ISSUE TITLE
V1.1.1 - Error when launching presets using python commands

### ISSUE THREAD

**Issue by jblanteigne** [DarienBrito/TDMorph#45]
basic command:
```
sliderContainer = op('TDMorph').GetContainer(1)
sliderContainer.SetUISetPreset(0)
```

I tried my command with V1.0 to make sure it works. When I send the same command to V1.1.1 it outputs the following error:
```
Traceback (most recent call last):
  File "</text2:op('/text2').run()>", line 1
td.Error: File "/text2", line 4
  File "/TDMorph/SlidersContainer1/PanelContent/SlidersContainer/extUIExtension", line 174, in SetUISetPreset
  File "/TDMorph/SlidersContainer1/PanelContent/SlidersContainer/extUIExtension", line 87, in setUIWidgetValue
AttributeError: 'NoneType' object has no attribute 'par'
Results of run operation resulted in exception.
```

**Comment by DarienBrito** [comment]
That's odd... not sure what broke this... will investigate.

**Comment by DarienBrito** [comment]
Fixed in V1.1.2
